src/utils/__tests__/sortPsp.test.ts

```diff
@@ -0,0 +1,57 @@
+import { sortBy } from "./../../utils/SortUtil";
+
+const bundles = [
+  { taxPayerFee: 100 },
+  { taxPayerFee: 200 },
+  { taxPayerFee: 150 },
+  { taxPayerFee: undefined },
+  { taxPayerFee: 0 },
+];
+
+test("sortBy function sorts bundles correctly in ascending order", () => {
+  const sorted = JSON.parse(JSON.stringify(bundles)).sort(
+    sortBy("taxPayerFee", "asc")
+  ); // Deep copy
+  expect(sorted).toEqual([
+    { taxPayerFee: 0 }, // 0 should come first
+    { taxPayerFee: 100 },
+    { taxPayerFee: 150 },
+    { taxPayerFee: 200 },
+    { taxPayerFee: undefined },
+  ]);
+});
+
+test("sortBy function places undefined values at the end in ascending order", () => {
+  const sorted = JSON.parse(JSON.stringify(bundles)).sort(
+    sortBy("taxPayerFee", "asc")
+  ); // Deep copy
+  expect(sorted[sorted.length - 1].taxPayerFee).toBeUndefined();
+});
+
+test("sortBy function sorts bundles correctly in descending order", () => {
+  const sorted = JSON.parse(JSON.stringify(bundles)).sort(
+    sortBy("taxPayerFee", "desc")
+  ); // Deep copy
+  expect(sorted).toEqual([
+    { taxPayerFee: 200 },
+    { taxPayerFee: 150 },
+    { taxPayerFee: 100 },
+    { taxPayerFee: 0 }, // 0 should come before undefined
+    { taxPayerFee: undefined },
```

Comment:
are we sure that there are cases where fees are undefined? this should never happen since it will mean that for that specific bundle we don't know the fee that will be applied to the transaction 🤔

Reply:
undefined is just a security measure just in case we have malformed data for any reason.

note that undefined will be always at the bottom of the list (both asc and desc) to avoid catch the user eye,
just in case

in general, better safe than sorry i guess.